### ISSUE TITLE
Missing @WebHealthCheck or @TCPHealthCheck doesn't clear the containers on error

### ISSUE THREAD

**Issue by DarthSeppius** [dockerunit/dockerunit-core#17]
**ISSUE**
If we are missing the @TCPHealthCheck or @WebHealthCheck on the descriptior definition we got an error saying:
```
java.lang.RuntimeException: No svc name detected. Make sure you have defined a health-check by using @WebHealthCheck or @TCPHealthCheck
```
but then the cleanup phase is not executed correctly because I still see the previously spawned containers before the actual failure.

We should check why the cleanup phase is not completed properly in this scenario.

DOCKERUNIT VERSION: `0.1.0`

**Comment by qzagarese** [comment]
I think we should fail fast if we don't detect a suitable annotation, given that the health-check is necessary

**Comment by irotech** [comment]
Yeah I noticed it as well we'd be cleaning up the docker images in case of error. I'd look into it.
